data_model/src/permission.rs

```diff
@@ -117,6 +125,57 @@ impl core::fmt::Display for PermissionToken {
     }
 }
 
+impl Encode for EncodableJson {
+    fn encode_to<W: parity_scale_codec::Output + ?Sized>(&self, dest: &mut W) {
+        self.0.to_string().encode_to(dest)
+    }
+
+    fn size_hint(&self) -> usize {
+        self.0.to_string().size_hint()
+    }
```

Comment:
Encoding JSON twice might be expensive. Although it is not critical now.

Reply:
Will address this under [this](https://github.com/hyperledger/iroha/pull/3775#discussion_r1287191647) discussion